src/foam/java/Enum.js

```diff
@@ -1,97 +1,64 @@
-// Test with: foam.u2.ControllerMode.model_.javaSource();
+// Defines template for the EnumModel
 
 foam.CLASS({
   refines: 'foam.core.EnumModel',
 
-  methods: [
-  ],
-
   templates: [
     {
       name: 'javaSource',
       template: `
 // Generated by FOAM, do not modify.
 
 package <%= this.package %>;
-
-public enum <%= this.name %> {
-  <%
-    for ( var i = 0 ; i < this.values.length ; i++ ) {
-      var value = this.values[i];
-  %>  <%= value.name %> <%
-    }
-  %>
-}
-      `
-    }
-  ]
-})
-
-/*
-package <%= this.package %>;
 <%
 var labelForValue = function(value) {
   if (this.androidResource) {
```

Comment:
Do we need this now? When we do, can we integrate it with FOAM's Message axiom support and then integrate the message support with Android Resources.